### DISCUSSION TITLE
List Thumbnail-Images of faces for one person

### DISCUSSION THREAD

**Discussion by Geekados** [photoprism/photoprism#4070] [Q&A]
Is it possible, to list all Thumbnail-Images for a single Person on one page?

Why I ask:
When i choose Photos of a person, there are always thumbnail-images with trees, the sky or walls, where the AI "found" a face (or even marked a wrong face).
So i started to click trough every image to check if the thumbnails are correct, this takes a lot of time.

It would be great, if I could choose "Images of 'me', show as Thumbnail-List", so i can scroll trough the list and drop/correct wrong ones.

Thanks for help!

**Comment by graciousgrey** [comment]
This is not possible yet. However something similar is planned for the future: https://github.com/photoprism/photoprism/issues/1601 :)

**Reply by Geekados** [reply]
Thanks für the quick reply!

This feature would be great and helpful! 

Until its finished*, I'll keep on clicking. :) 
_*or I reached the end of my motivation... :)_
